bot/exts/fun/off_topic_names.py

```diff
@@ -110,11 +114,60 @@ async def _add_name(self, ctx: Context, name: str) -> None:
     @has_any_role(*MODERATION_ROLES)
     async def delete_command(self, ctx: Context, *, name: OffTopicName) -> None:
         """Removes a off-topic name from the rotation."""
+        await self._delete_name(ctx, name)
+
+    async def _delete_name(self, ctx: Context, name: str) -> None:
+        """Delete an off-topic channel name."""
         await self.bot.api_client.delete(f'bot/off-topic-channel-names/{name}')
 
         log.info(f"{ctx.author} deleted the off-topic channel name '{name}'")
         await ctx.send(f":ok_hand: Removed `{name}` from the names list.")
 
+    @otname_group.command(name='reroll')
+    @has_any_role(*MODERATION_ROLES)
+    async def re_roll_command(self, ctx: Context, ot_channel_index: Optional[int]) -> None:
```

Comment:
Why make `ot_channel_index` optional if we're gonna raise an error if it wasn't passed?

Reply:
If it is invoked in an OT channel then it will reroll that channel's name.

Reply:
smart, didn't notice that part